pandas/tests/io/json/test_normalize.py

```diff
@@ -287,6 +288,30 @@ def test_shallow_nested(self):
         expected = DataFrame(ex_data, columns=result.columns)
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.skipif(sys.version_info < (3, 6), reason="drop support for 3.5 soon")
+    def test_nested_meta_path_with_nested_record_path(self, state_data):
+        # GH 27220
+        result = json_normalize(
+            state_data,
```

Comment:
can you use keywords here, this is very hard to read

Reply:
Sure. Good suggestion~